src/evm_context.rs

```diff
@@ -168,6 +189,26 @@ pub fn mload(index: &String, ssa: &mut SSATracker) -> String {
     format!("(concat {})", arguments)
 }
 
+pub fn keccak256(offset: &String, length: &String, ssa: &mut SSATracker) -> String {
+    let scratch_0 = mload(&smt::SMTFormat::as_smt(&0), ssa);
+    let scratch_1 = mload(&smt::SMTFormat::as_smt(&0x20), ssa);
+    smt::ite(
+        &smt::and(&smt::eq(offset, &0), &smt::eq(length, &0x20)),
```

Comment:
Do we need to check whether offset is 0? Could also just `mload` from whatever offset is as long as the length is correct, and hope the SMT solver's constant propagation takes care of the 0 here

Reply:
Oh right, that is totally true!